scripts/generate-version-info.sh

```diff
@@ -16,8 +16,17 @@ OUTPUT_FILE="$1"
 sudo rpm --query --all --queryformat '\{"%{NAME}": "%{VERSION}-%{RELEASE}"\}\n' | jq --slurp --sort-keys 'add | {packages:(.)}' > "$OUTPUT_FILE"
 
 # binaries
-echo $(jq ".binaries.kubelet = \"$(kubelet --version | awk '{print $2}')\"" $OUTPUT_FILE) > $OUTPUT_FILE
-echo $(jq ".binaries.awscli = \"$(aws --version | awk '{print $1}' | cut -d '/' -f 2)\"" $OUTPUT_FILE) > $OUTPUT_FILE
+KUBELET_VERSION=$(kubelet --version | awk '{print $2}')
+if [ "$?" != 0 ]; then
```

Comment:
Will we ever get here since the script is set to fail on errors?

Reply:
script doesn't fail, that's the problem. (also see above)

Reply:
Right, but if it gets here, it's still not going to fail, right?

Reply:
sorry missed the exit (didn't push it)